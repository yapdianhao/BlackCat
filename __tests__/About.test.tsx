import React from "react";
import * as renderer from "react-test-renderer";

import About from "../components/About";

it("renders About correctly", () => {
  const tree = renderer.create(<About />).toJSON();
  expect(tree).toMatchSnapshot();
});
