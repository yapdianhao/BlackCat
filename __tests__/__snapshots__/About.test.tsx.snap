// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders About correctly 1`] = `
<div>
  About
</div>
`;
